src/property-interpolation.js

```diff
@@ -90,10 +90,13 @@
   };
 
   function propertyInterpolation(property, left, right) {
+    var ucProperty = property.replace(/-(.)/g, function(_, c) {
+      return c.toUpperCase();
+    });
+    if (ucProperty !== property) {
+      shared.deprecated('Hyphenated property names', '2016-03-22', 'Use camelCase instead.', true);
```

Comment:
I manually checked in Firefox and Chromium ToT.

For three months, hyphenated property names will animate with the polyfill deprecation message. (No Blink deprecation message.)

After three months, hyphenated property names will flip. (No Blink or Polyfill deprecation message.)